client/main.go

```diff
@@ -43,81 +102,44 @@ func usableHostName(orig string) (valid string) {
 	return
 }
 
-func newFuzzer() Fuzzer {
+func newAFLFuzzer() Fuzzer {
 	name, err := os.Hostname()
 	if err != nil {
 		log.Fatal("Couldn't get hostname", err)
 	}
 
 	name = usableHostName(name)
 	number := types.RandInt() & 0xffff
+	id := fmt.Sprintf("%s-%x", name, number)
+
+	memLimit := os.Getenv("AFL_MEMORY_LIMIT")
```

Comment:
I'm not positive that environment variables are how we want to hand these values around. I think I'd nominally prefer cli flags until that gets too unwieldy and then probably a config file?

Reply:
I'm happy to do whatever - I copied this over from https://github.com/richo/roving/pull/5/files#diff-c9eff17208b30aa160325bc597269098L77 ;)

Reply:
ahaha. Classic past-richo. Feel free to leave it.